ci/scripts/cron-e2e-test.sh

```diff
@@ -1,8 +0,0 @@
-#!/usr/bin/env bash
-
-# Exits as soon as any line fails.
-set -euo pipefail
-
-source ci/scripts/common.sh
-export RUN_COMPACTION=0;
```

Comment:
It seems we forgot to re-enable it.
https://github.com/risingwavelabs/risingwave/pull/14367#discussion_r1442466100